elasticsearch/Dockerfile

```diff
@@ -37,6 +37,7 @@ RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
 
 ADD sgconfig/ ${HOME}/sgconfig/
 ADD index_templates/ /usr/share/elasticsearch/index_templates/
+ADD probe/readiness.sh /usr/share/elasticsearch/probe/readiness.sh
```

Comment:
i'm not 100% but you may need to create the `probe` sub-directory first... travis should verify though